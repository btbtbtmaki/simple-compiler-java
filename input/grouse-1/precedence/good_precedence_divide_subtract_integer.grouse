main {

  print 93 / 37 - 37 nl;
  print 74 / 70 - 59 nl;
  print 17 / 68 - 24 nl;
  print 67 / 51 - 83 nl;
  print 97 / 54 - 69 nl;
  print 89 / 33 - 23 nl;
  print 26 / 86 - 57 nl;
  print 31 / 19 - 29 nl;
  print 80 / 19 - 68 nl;
  print 26 / 94 - 93 nl;
  print 70 / 5 - 87 nl;
  print 1 / 59 - 26 nl;

  print 75 - 54 / 68 nl;
  print 14 - 81 / 77 nl;
  print 42 - 58 / 68 nl;
  print 43 - 90 / 60 nl;
  print 48 - 34 / 68 nl;
  print 97 - 37 / 13 nl;
  print 16 - 27 / 58 nl;
  print 15 - 89 / 13 nl;
  print 44 - 11 / 85 nl;
  print 90 - 36 / 27 nl;
  print 61 - 58 / 34 nl;
  print 39 - 59 / 21 nl;

  print 26 / 51 - 88 / 37 nl;
  print 32 / 99 - 72 / 47 nl;
  print 9 / 87 - 48 / 80 nl;
  print 84 / 95 - 18 / 66 nl;
  print 85 / 8 - 57 / 61 nl;
  print 53 / 2 - 63 / 56 nl;
  print 62 / 42 - 95 / 6 nl;
  print 56 / 55 - 91 / 1 nl;
  print 24 / 1 - 72 / 88 nl;
  print 49 / 47 - 29 / 33 nl;
  print 80 / 54 - 1 / 94 nl;
  print 63 / 63 - 82 / 53 nl;

  print 48 - 79 / 95 - 79 nl;
  print 13 - 18 / 49 - 5 nl;
  print 96 - 57 / 39 - 17 nl;
  print 69 - 87 / 42 - 95 nl;
  print 85 - 98 / 52 - 14 nl;
  print 26 - 53 / 5 - 92 nl;
  print 38 - 57 / 71 - 83 nl;
  print 49 - 67 / 81 - 46 nl;
  print 51 - 89 / 98 - 72 nl;
  print 14 - 97 / 64 - 57 nl;
  print 20 - 54 / 75 - 50 nl;
  print 24 - 47 / 57 - 59 nl;

}

