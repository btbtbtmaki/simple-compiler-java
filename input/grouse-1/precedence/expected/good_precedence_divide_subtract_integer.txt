-35
-58
-24
-82
-68
-21
-57
-28
-64
-93
-73
-26
75
13
42
42
48
95
16
9
44
89
60
37
-2
-1
0
0
10
25
-14
-90
24
1
1
0
-31
8
78
-28
70
-76
-45
3
-21
-44
-30
-35
